src/App.tsx

```diff
@@ -1,58 +1,70 @@
-import React, { useState } from "react";
-import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import React, { useState } from 'react'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
 import {
   setAccessTokenCookie,
   signOut,
-} from "synapse-react-client/dist/utils/SynapseClient";
-import "./App.scss";
-import AppInitializer from "./AppInitializer";
-import { OAuth2Form } from "./OAuth2Form";
-import { getURLParam } from "./URLUtils";
-import Versions from "./Versions";
+} from 'synapse-react-client/dist/utils/SynapseClient'
+import './App.scss'
+import AppInitializer from './AppInitializer'
+import { OAuth2Form } from './OAuth2Form'
+import { getURLParam } from './URLUtils'
+import Versions from './Versions'
+import generalTheme from './style/theme'
+import {
+  createTheme,
+  StyledEngineProvider,
+  ThemeProvider,
+} from '@mui/material/styles'
 
 const App: React.FC = () => {
-  const [isLoggedOut, setIsLoggedOut] = useState(false);
+  const [isLoggedOut, setIsLoggedOut] = useState(false)
+  const theme = createTheme(generalTheme)
+  debugger
   return (
     <div className="App">
-      <Router>
```

Comment:
Wrap to support theme.  Theme definition palette uses the base synapse palette defined in SRC